templates/CRM/Report/Form/Layout/Table.tpl

```diff
@@ -99,7 +99,7 @@
                     {assign var=fieldClass value=$field|cat:"_class"}
                     <td class="crm-report-{$field}{if $header.type eq 1024 OR $header.type eq 1 OR $header.type eq 512} report-contents-right{elseif $row.$field eq 'Subtotal'} report-label{/if}">
                         {if array_key_exists($fieldLink, $row) && $row.$fieldLink}
-                            <a title="{$row.$fieldHover|escape}" href="{$row.$fieldLink}"  {if array_key_exists($fieldClass, $row)} class="{$row.$fieldClass}"{/if}>
+                            <a title="{$row.$fieldHover|default:''|escape}" href="{$row.$fieldLink}"  {if array_key_exists($fieldClass, $row)} class="{$row.$fieldClass}"{/if}>
```

Comment:
Using `default` isn't compatible with the escape-by-default setting (the error still appears). Also while it doesn't come up in core currently, if unit tests were made louder this might still error because `@` gets ignored, and default uses `@` [behind the scenes](https://lab.civicrm.org/dev/core/-/issues/2795#smarty-default-modifier).

There are some draft docs at https://lab.civicrm.org/documentation/docs/dev/-/issues/880 for a couple things we've been trying to wrangle these. Maybe in this case since it's a different fieldname each time it might be easiest to do something like this?

`<a {if array_key_exists($fieldHover, $row)}title="{$row.$fieldHover|escape}"{/if} href="{$row.$fieldLink}"  {if array_key_exists($fieldClass, $row)} class="{$row.$fieldClass}"{/if}>`

Reply:
lol